examples/demo10.yaml

```diff
@@ -100,12 +103,18 @@ gal :
             # However, sequences are indexed by the overall object number, so in order to
             # step through the items in this list once for each time it gets to the "first"
             # item, we need to set the index to repeat 20 times before incrementing.
-            # We again use the convenient YAML variable name notation to do so.
-            index : { type : Sequence , repeat : *num_in_ring }
+            index :
+                type : Sequence
+                # You can access the current value of any other item in the config dict using
+                # type = Current.  The syntax for the key is to treat the config levels as
+                # attributes.  So the number of galaxies in the ring defined above is gal.num.
+                # You can use any value this way.  If it hasn't actually been evaluated yet,
+                # it will evaluate it as needed for the current object.
+                repeat : { type : Current, key : gal.num }
```

Comment:
Does the old raw-YAML approach work too?  If so, is there an advantage to using one or the other, besides readability?